.github/workflows/go.yml

```diff
@@ -0,0 +1,132 @@
+        uses: percona-platform/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: ${{ env.WORKING_DIR }}
+
+      - name: Set up Go release
+        if: matrix.go.version != 'tip'
+        uses: percona-platform/setup-go@v2
+        with:
+          go-version: ${{ matrix.go.version }}
+
+      - name: Set up Go tip
+        if: matrix.go.version == 'tip'
+        run: |
+          git clone --depth=1 https://go.googlesource.com/go $HOME/gotip
+          cd $HOME/gotip/src
+          ./make.bash
+          echo "GOROOT=$HOME/gotip" >> $GITHUB_ENV
+          echo "$HOME/gotip/bin" >> $GITHUB_PATH
+
+      - name: Enable Go modules cache
+        uses: percona-platform/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ matrix.os }}-go-${{ matrix.go-version }}-modules-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ matrix.os }}-go-${{ matrix.go-version }}-modules-
+
+      - name: Download Go modules
+        run: go mod download
+
+      - name: API make Init
+        run: make -C api init
+
+      - name: API Prepare
+        run: |
+          # Ensure that generated files are not changed
+          make -C api gen gen-alertmanager
+          git status
+          git diff --exit-code
+
+      - name: Update API compatibility descriptors.
+        run: |
+          # log if descriptors changed, useful for "update descriptors" PRs
+          make -C api descriptors
+          git diff --text
+
+      - name: Run tests
+        run: go test ./...
+
+      - name: Clean Testcache
+        run: go clean -testcache
+
+  workflow_success:
+    needs: [build]
+    name: Slack Notification success
+    runs-on: ubuntu-latest
+    env:
+      SLACK_WEBHOOK: ${{ secrets.SLACK_TOKEN }}
+      SLACK_CHANNEL: "pmm-ci"
+      SLACK_USERNAME: "PR pipelines bot"
+      SLACK_ICON_EMOJI: ":chestnut:"
+      SLACK_COLOR: "#00FF00"
+      SLACK_TITLE: "Finished ${{ github.event.repository.name }} workflow"
+      SLACK_MESSAGE: "${{ github.event.inputs.repo || github.repository }}:${{ github.event.inputs.branch || github.head_ref }}"
+      SLACK_FOOTER: "Please check the Actions URL ^"
+
+    steps:
+      - uses: percona-platform/checkout@v2
+      - uses: percona-platform/setup-node@v2
+      - name: Slack Notification
+        uses: percona-platform/action-slack-notify@v2
+
+  workflow_failure:
+    if: ${{ failure() }}
+    needs: [build]
+    name: Slack Notification failure
+    runs-on: ubuntu-latest
+    env:
+      SLACK_WEBHOOK: ${{ secrets.SLACK_TOKEN }}
+      SLACK_CHANNEL: "pmm-ci"
+      SLACK_USERNAME: "PR pipelines bot"
+      SLACK_ICON_EMOJI: ":chestnut:"
+      SLACK_COLOR: "#FF0000"
+      SLACK_TITLE: "Finished ${{ github.event.repository.name }} workflow"
+      SLACK_MESSAGE: "Workflow failed: ${{ github.event.inputs.repo || github.repository }}:${{ github.event.inputs.branch || github.head_ref }}"
+      SLACK_FOOTER: "Please check the Actions URL ^"
```

Comment:
Can we remove `SLACK_FOOTER: Please check the Actions URL ^`? Looks redundant and just clutters the message.